pkg/document/document.go

```diff
@@ -1,60 +1,298 @@
 package document
 
-import sitter "github.com/smacker/go-tree-sitter"
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	sitter "github.com/smacker/go-tree-sitter"
+	"go.lsp.dev/protocol"
+	"go.lsp.dev/uri"
+
+	"github.com/tilt-dev/starlark-lsp/pkg/query"
+)
+
+type LoadStatement struct {
+	File        string
+	Vars        [][2]string
```

Comment:
"Symbols" might be better here - it's easy to read "Vars" as excluding functions

Reply:
Yeah, I like that better